docs/fsharp/what-is-fsharp.md

```diff
@@ -1,31 +1,29 @@
 ---
 title: What is F#
-description: Learn about what the F# programming language is and what F# programming is like. Learn about rich data types, functions, and how they fit together.
+description: Learn about what the F# programming language is and what F# programming is like. Learn about rich types, functions, and how they fit together.
 ms.date: 08/03/2018
 ---
 # What is F\#
 
-F# is a functional programming language that makes it easy to write correct and maintainable code.
+F# is an open-source, cross-platform, interoperable programming language for writing succinct, robust and performant code.
```

Comment:
I think that, given this is docs and not a marketing site, mentioning functional here actually does make sense. The rest of the sentence does feel like the right framing though.

Reply:
It's the landing page for a lot of "What is F#" links so we would like this core description to be the start. We can then explain the technical characteristics.  

"Functional" will in general be de-emphasised across the docs slowly.  Data+Functions will be emphasised in replacement, per https://twitter.com/isaac_abraham/status/1417122528607227906/photo/1 :)